lbz/dev/event.py

```diff
@@ -0,0 +1,52 @@
+# flake8: noqa
+EVENT = """
```

Comment:
maybe `SAMPLE_EVENT`? @redlickigrzegorz

Reply:
Or `EVENT_TEMPLATE` 🤔  but yes, we can improve this name.